NeuralNetworks.ipynb

```diff
@@ -128,19 +128,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Please change the runtime to GPU if you'd like to have some speed-up on Colab\n",
+    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
```

Comment:
Just checking, does this code run locally as well, or do students need to configure stuff to make this work? Cause I can update the README with instructions on this